conformance/report_template.gohtml

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<style>
+    table, th, td {
+        border: 1px solid black;
+    }
+    td {
+        padding: 2px;
+    }
+</style>
+<body>
+<h2>MCS Conformance Report</h2>
+
+{{range .Groups}}
+<h3>{{.Name}}</h3>
+<table>
+    <th>Result</th>
+    <th>Description</th>
```

Comment:
This part of my previous suggestion is required for the table to be valid HTML; in fact that wasn’t complete:

```suggestion
    <thead>
        <tr>
            <th>Result</th>
            <th>Description</th>
        </tr>
    </thead>
```

Reply:
It's been a long time since I've written HTML. I actually grabbed this code as a base from the subctl summary HTML. As is, it works on Chrome and Firefox but perhaps it's deprecated syntax.

Reply:
> It's been a long time since I've written HTML. I actually grabbed this code as a base from the subctl summary HTML. As is, it works on Chrome and Firefox but perhaps it's deprecated syntax.

Yeah it works fine, I just like having valid HTML :wink: